client/src/pages/HomePage.js

```diff
@@ -0,0 +1,169 @@
+import Avatar from "../components/Avatar";
+import { Link } from "react-router-dom";
+import Plural from "../components/Plural";
+import FightPilette from "../images/fight.png";
+import WaitPilette from "../images/attente.png";
+
+class HomePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentUser: null,
+      usersData: [],
+      toPlayChallenges: [],
+      pendingChallenges: [],
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("/api/v1/duels/")
+      .then((res) => {
+        const toPlay = [],
+          pending = [];
+        res.data.forEach((val) => {
+          if (!val.inProgress) return;
+          if (val.rounds[val.currentRound - 1][0].userAnswer !== undefined) {
+            pending.push(val);
+          } else {
+            toPlay.push(val);
+          }
+        });
+        this.setState({
+          toPlayChallenges: toPlay,
+          pendingChallenges: pending,
+        });
+
+        const listOfUsers = [
+          ...toPlay.map((value) => value.opponent),
+          ...pending.map((value) => value.opponent),
+        ];
+        this.getUsersData(listOfUsers);
+      })
+      .catch((err) => console.error(err));
+  }
+
+  getUsersData(otherUsers) {
+    const currentUser = AuthService.getCurrentUser();
+    const listOfUsers = [currentUser.pseudo, ...otherUsers];
+    axios
+      .post("/api/v1/users/", listOfUsers)
+      .then((res) => {
+        this.setState({
+          currentUser: res.data[currentUser.pseudo],
+          usersData: res.data,
+        });
+      })
+      .catch((err) => console.error(err));
+  }
+
+  render() {
+    const { currentUser, usersData, toPlayChallenges, pendingChallenges } = this.state;
+    const cuWins = currentUser?.wins ?? 0;
+    const cuLosses = currentUser?.losses ?? 0;
+
+    return (
+      <main id="homepage">
+        <header>
+          <div id="header-background"></div>
+          <Avatar
+            size="125px"
+            eyes={currentUser?.avatar.eyes ?? 0}
+            hands={currentUser?.avatar.hands ?? 0}
+            hat={currentUser?.avatar.hat ?? 0}
+            mouth={currentUser?.avatar.mouth ?? 0}
+            colorBG={currentUser?.avatar.colorBG ?? "#d3d3d3"}
+            colorBody={currentUser?.avatar.colorBody ?? "#0c04fc"}
+          />
+
+          <div>
+            <h1>{currentUser?.pseudo ?? "Pilette"}</h1>
+            <p>
+              {cuWins} <Plural word="victoire" count={cuWins} />
+            </p>
+            <p>
+              {cuLosses} <Plural word="défaite" count={cuLosses} />
+            </p>
+          </div>
+        </header>
+
+        <div id="links">
+          <Link to="/train" className="btn">
+            Entraînement
+          </Link>
+          <Link to="/homepage" className="btn">
+            Nouveau duel
+          </Link>
+        </div>
+
+        <section>
+          <h2>
+            <img src={FightPilette} alt="Pilette is fighting" /> Ton tour
```

Comment:
Shouldn't alt texts be in French as all of our UI is in French?

Reply:
Oh yes!